lib/views/pages/newsfeed/newsArticle.dart

```diff
@@ -103,32 +110,32 @@ class _NewsArticleState extends State<NewsArticle> {
                 leading: CircleAvatar(
                   backgroundImage: AssetImage(UIData.pkImage),
                 ),
-                title:  Container(
+                title: Container(
                   constraints: BoxConstraints(
                     maxHeight: double.infinity,
                     minHeight: 20,
                   ),
-                child: TextField(
-                  textInputAction:  TextInputAction.newline,
-                  keyboardType: TextInputType.multiline,
-                  //minLines: 1,//Normal textInputField will be displayed
-                  //maxLines: 10,// when user presses enter it will adapt to it
-                  maxLines: null,
-                  decoration: InputDecoration(
-                      suffix: IconButton(
-                        color: Colors.grey,
-                        icon: Icon(Icons.send),
-                        onPressed: () {
-                          print(commentController.text);
-                          createComment();
-                        },
-                      ),
-                      hintText: 'Leave a Comment....',
-                      border: OutlineInputBorder(
-                          borderRadius: BorderRadius.circular(20.0),
-                          borderSide: BorderSide(color: Colors.teal))),
-                  controller: commentController,
-                ),
+                  child: TextField(
```

Comment:
@utkarshshendge is this change required to fix the issue ?

Reply:
No @Sagar2366 Sir, looks like its result of whitespace I gave.
Should I create another commit?

Reply:
Yes, it clearly shows the effect of this in second video.